docs/low-code-spark/execution/databricks-clusters-behaviours.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Execution on Databricks
+id: executions_on_databricks_clusters
+description: Execution on Databricks clusters
+sidebar_position: 3
+tags:
+  - execution
+  - interim
+  - UnityCatalog
+  - Shared
+  - metrics
+  - spark
+---
+
+Databricks cluster comes with various [Access Modes](https://docs.databricks.com/clusters/create-cluster.html#what-is-cluster-access-mode).
+Depending on the cluster type, external library installations might be restricted on some clusters as mentioned below.
+
+For [Interims](./interactive-execution#interims) and [Execution Metrics](./execution-metrics) to work, Prophecy has written some libraries in Python and Scala, which needs to be installed on cluster. Hence, because of below restrictions, interims behaviour will be different for different clusters.
+
+| Access mode dropdown      | Library Installations Allowed?          | Language supported | Interims             | Execution Metrics |
+| ------------------------- | --------------------------------------- | ------------------ | -------------------- | ----------------- |
+| Single User               | Yes                                     | Python, Scala      | Regular Interims     | Yes               |
+| Shared                    | Yes, but interim nodes cant be attached | Python             | **Vanilla Interims** | **No**            |
+| No isolation shared       | Yes                                     | Python, Scala      | Regular Interims     | Yes               |
+| Unity Catalog Single User | Yes                                     | Python, Scala      | Regular Interims     | Yes               |
+| Unity Catalog Shared      | No                                      | Python             | **Vanilla Interims** | **No**            |
+
+### Interims
+
+During development, often the user will want to see their data to make more sense of it and to check whether the expected output is getting
+generated or not after the transformation. Prophecy generates these data samples as `Interims`, which are temporarily cached previews of data after each Gem.
+To check more about interims, please refer [here](./interactive-execution#interims)
+
+Depending on the type of clusters, we have two modes of Interims
+
+#### Regular Interims
+
+For Single User clusters, and No isolation Shared clusters, we have interims available after each Gem of Pipeline. These are available on both Unity and Non Unity catalog workspaces.
+
+![Regular Interims](img/SingleModeIntermisAndExcutionMetrics.png)
+
+#### Vanilla Interims
+
+For Shared mode clusters, since we cannot attach interims after each gems, we have added interims on all Actions/Tail nodes of the Pipeline.
+Please see the below image for reference.
+
+![Vanilla Interims](img/SharedModeInterims.png)
```

Comment:
2 things with this
1. Vanilla interims are shown differently for non-Target nodes. Target nodes show vanilla interims before them, rest all show vanilla interims after them. Just execute a pipeline without Target and multiple end (dangling) nodes to see the difference
2. Vanilla interims are not shown like this anymore. They have been changed from "floating" to on the edge

Reply:
Updated the docs with new screenshots.